symbulate/distributions.py

```diff
@@ -135,14 +135,14 @@ def __init__(self, n, N0, N1):
         if n > 0 and (isinstance(n, int)):
             self.n = n
         else:
-            raise Exception("Number of draws must be an integer and cannot be negative")
+            raise Exception("Number of draws must be a non-zero integer and cannot be negative")
         
-        if N0 > 0 and (isinstance(N0, int)):
+        if N0 >= 0 and (isinstance(N0, int)):
             self.N0 = N0
         else:
             raise Exception("Number of failures must be an integer and cannot be negative")
```

Comment:
It's not accurate to call this "number of failures". Say "number of 0s" to match the docstring.